src/main/java/org/shanerx/tradeshop/Utils.java

```diff
@@ -385,6 +475,22 @@ public ItemStack isValidType(String mat, int durability, int amount) {
     }
 
     /**
+     * Checks whether or not it is a valid material or custom item.
+     *
+     * @param itm Item to check
+     * @return true if item is blacklist item
+     */
+    public boolean isBlacklistItem(ItemStack itm) {
```

Comment:
I prefer the name itemIstBlacklisted() as it sounds more natural to me. Just taste though.